docs/operation/cluster_and_service_configuration.rst

```diff
@@ -0,0 +1,249 @@
+
+#. Save and apply the changes.
+
+#. Connect to the Salt master container using:
+
+   .. code-block:: shell
+
+      root@bootstrap $ crictl exec -it \
+                         $(crictl ps -q --label io.kubernetes.container.name=salt-master) bash
+
+#. From the Salt master container, apply the following salt-states to propagate
+   the new changes down to the underlying services.
+
+   .. code-block:: shell
+
+      $ salt-run state.sls metalk8s.addons.dex.deployed saltenv=metalk8s-<version>
+
+#. Finally, create and apply the required :file:`ClusterRoleBinding.yaml` file
+   that ensures that the newly added static user is bound to a Cluster Role.
+
+   .. note::
+
+      MetalK8s installations come with already existing Cluster Roles.
+      Administrators can create new Cluster Roles or refer to the existing
+      Cluster Roles.
+
+      To obtain the list of available Cluster Roles in a MetalK8s cluster,
+      use the following command:
+
+      .. code-block:: shell
+
+         root@bootstrap $ kubectl --kubeconfig /etc/kubernetes/admin.conf get clusterroles
+
+      For more information about a Cluster Role, run the following command to
+      describe it.
+
+      .. code-block:: shell
+
+         root@bootstrap $ kubectl --kubeconfig /etc/kubernetes/admin.conf get clusterroles \
+                            <name> -o yaml
+
+      For starters, MetalK8s administrators can provision new users using the
+      `cluster-admin` Cluster Role. Note that this Cluster Role by default
+      grants cluster-wide permissions to all resources within a cluster.
+      For more information refer to
+      `RBAC <https://kubernetes.io/docs/reference/access-authn-authz/rbac/>`_
+      documentation.
+
+
+   - Use the following template to create the :file:`ClusterRoleBinding.yaml`
+     file where:
+
+      - <name> refers to any freely chosen name
+      - <email> refers to the new user email
+      - <cluster-role> refers to the Cluster Role picked from the list above
+
+   .. code-block:: yaml
+
+      apiVersion: rbac.authorization.k8s.io/v1
+      kind: ClusterRoleBinding
+      metadata:
+        name: <name>
+      subjects:
+      - kind: User
+        name: <email>
+        apiGroup: rbac.authorization.k8s.io
+      roleRef:
+        kind: ClusterRole
+        name: <cluster-role>
+        apiGroup: rbac.authorization.k8s.io
+
+   - Apply the ClusterRoleBinding configurations using:
+
+   .. code-block:: shell
+
+      root@bootstrap $ kubectl --kubeconfig /etc/kubernetes/admin.conf apply -f \
+                         ClusterRoleBinding.yaml
+
+#. Verify that the user has been successfully added and you can log in to the
+   MetalK8s UI using the new email and password.
+
+.. todo::
+
+   Add documentation on the following tracked topics
+
+   - Change static user password (issue #2075)
+
+   - External authentication (issue #2013)
+
+      - Configuring LDAP
+      - Configuring Active Directory(AD)
+
```

Comment:
Is it something that we can do from the GUI (create user, role ...) ?

Reply:
I am afraid no. cc @thomasdanan

Reply:
@xaf-scality UI configuration is tracked with #1991